Revit_Core_Engine/Query/Passes.cs

```diff
@@ -199,7 +199,7 @@ public static bool Passes(this Parameter parameter, FilterByParameterBool reques
             if (!CheckIfNotNull(parameter, request))
                 return false;
 
-            if (parameter.HasValue && parameter.StorageType == StorageType.Integer && parameter.Definition.ParameterType == ParameterType.YesNo)
+            if (parameter.StorageType == StorageType.Integer && parameter.Definition.ParameterType == ParameterType.YesNo)
```

Comment:
I would not agree with this change - no value is not equal to `false`. After your change, the empty parameter will pass a request asking for values equal to `false`, which is not true. However, correct me if I am wrong: this will never occur in practice anyways, taken that yes/no can't be empty in Revit, right?

Reply:
Hmm, if it was an academic discussion, I would argue that not checked != unchecked 🐒 But I understand your approach from the user's perspective, so happy to agree with you 👍